Makefile

```diff
@@ -140,8 +142,10 @@ contributors:
 	git log --format='%aN <%aE>' | sort -fu > CONTRIBUTORS
 
 js:
+ifeq (go1.8, $(findstring go1.8, $(GO_VERSION)))
```

Comment:
Thanks thats really good to know.
That also explains why there differences in the gzipped filedescriptor :)